homeassistant/components/leneda/const.py

```diff
@@ -0,0 +1,139 @@
+    },
+    "electricity_consumption_covered_layer4": {
+        "obis_code": ObisCode.ELEC_CONSUMPTION_COVERED_LAYER4,
+        "icon": "mdi:lightning-bolt",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+    "electricity_consumption_remaining": {
+        "obis_code": ObisCode.ELEC_CONSUMPTION_REMAINING,
+        "icon": "mdi:lightning-bolt",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+    # Electricity Production
+    "electricity_production_active": {
+        "obis_code": ObisCode.ELEC_PRODUCTION_ACTIVE,
+        "icon": "mdi:solar-power",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+    "electricity_production_reactive": {
+        "obis_code": ObisCode.ELEC_PRODUCTION_REACTIVE,
+        "icon": "mdi:solar-power",
+        "device_class": None,  # Ideally SensorDeviceClass.REACTIVE_POWER per hour
+        "state_class": "total_increasing",
+    },
+    "electricity_production_shared_layer1": {
+        "obis_code": ObisCode.ELEC_PRODUCTION_SHARED_LAYER1,
+        "icon": "mdi:solar-power",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+    "electricity_production_shared_layer2": {
+        "obis_code": ObisCode.ELEC_PRODUCTION_SHARED_LAYER2,
+        "icon": "mdi:solar-power",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+    "electricity_production_shared_layer3": {
+        "obis_code": ObisCode.ELEC_PRODUCTION_SHARED_LAYER3,
+        "icon": "mdi:solar-power",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+    "electricity_production_shared_layer4": {
+        "obis_code": ObisCode.ELEC_PRODUCTION_SHARED_LAYER4,
+        "icon": "mdi:solar-power",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+    "electricity_production_remaining": {
+        "obis_code": ObisCode.ELEC_PRODUCTION_REMAINING,
+        "icon": "mdi:solar-power",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+    # Gas Consumption
+    "gas_consumption_volume": {
+        "obis_code": ObisCode.GAS_CONSUMPTION_VOLUME,
+        "icon": "mdi:fire",
+        "device_class": "gas",
+        "state_class": "total_increasing",
+    },
+    "gas_consumption_standard_volume": {
+        "obis_code": ObisCode.GAS_CONSUMPTION_STANDARD_VOLUME,
+        "icon": "mdi:fire",
+        "device_class": None,  # Ideally SensorDeviceClass.GAS, but Nm3 not supported by Hass
+        "state_class": "total_increasing",
+    },
+    "gas_consumption_energy": {
+        "obis_code": ObisCode.GAS_CONSUMPTION_ENERGY,
+        "icon": "mdi:fire",
+        "device_class": "energy",
+        "state_class": "total_increasing",
+    },
+}
+
+UNIT_TO_AGGREGATED_UNIT = {"kW": "kWh", "kVAR": "kVARh"}
+
+# Integration description
+INTEGRATION_DESCRIPTION = """
```

Comment:
This seems like its something for the docs rather than in the code? I might have missed something though because GitHub hasn't loaded all the files.

Reply:
Thanks, this was indeed not supposed to be in there. Removed.